OpenAI-DotNet/Chat/ChatRequest.cs

```diff
@@ -200,5 +219,17 @@ public sealed class ChatRequest
         /// </summary>
         [JsonPropertyName("user")]
         public string User { get; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [JsonPropertyName("functions")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public IReadOnlyList<Function> Functions { get; }
+
+        [JsonPropertyName("function_call")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public string FunctionCall { get; }
```

Comment:
let's move this up ahead of the Functions collection

Reply:
ok